tests/transports-integrations/Makefile

```
@@ -0,0 +1,117 @@
+# Development commands
+lint:
+	@echo "Running flake8..."
+	-flake8 *.py
+	@echo "Running mypy..."
+	-mypy *.py
+
+format:
+	@echo "Formatting code with black..."
+	black *.py
+
+clean:
```

checkmake (0.2.2)

[warning] Target body for "clean" exceeds allowed length of 5 (8).

(maxbodylength)